underlay/src/main/resources/config/display/sd/prepackagedcriteria/_demographics/demographics.json

```diff
@@ -0,0 +1,5 @@
+{
+  "entities": [
+    ""
```

Comment:
Can this be `person` instead of empty string?

Reply:
We can do it in a separate PR.

Reply:
OK, are you using this for anything other than filter building in the UI?

Reply:
No